src/sharedServices/bulkJobs/startGlacierRestore.ts

```diff
@@ -27,17 +27,13 @@ export const startGlacierRestore = async (
     filesToRestore,
     manifestFileName
   )
-  logger.info(
-    `Starting Glacier restore for zendesk ticket with id '${zendeskTicketId}'`
-  )
+  logger.info(`Starting Glacier restore`)
   const jobId = await createBulkGlacierRestoreJob(
     manifestFileName,
     manifestFileEtag,
     zendeskTicketId
   )
-  logger.info(
-    `Started Glacier restore for zendesk ticket with id '${zendeskTicketId}', with jobId '${jobId}'`
-  )
+  logger.info(`Started Glacier restore with jobId '${jobId}'`)
```

Comment:
Why are we removing the ticket id here? Thought that was good for traceability

Reply:
Because we're now putting the zendeskId into all logs as a property. That was the actually the original reason we put PowerLogger into the system, because it allows context logging (i.e. a property that follows through multiple logs without having to repeat it)